packages/react-scripts/scripts/utils/verifyTypeScriptSetup.js

```diff
@@ -172,13 +172,16 @@ function verifyTypeScriptSetup() {
 
     parsedCompilerOptions = result.options;
   } catch (e) {
-    console.error(
-      chalk.red.bold(
-        'Could not parse',
-        chalk.cyan('tsconfig.json') + '.',
-        'Please make sure it contains syntactically correct JSON.'
-      )
-    );
+    if (e && e.name === 'SyntaxError') {
+      console.error(
+        chalk.red.bold(
+          'Could not parse',
+          chalk.cyan('tsconfig.json') + '.',
+          'Please make sure it contains syntactically correct JSON.'
+        )
+      );
+    }
+    
     console.error(e && e.message ? `Details: ${e.message}` : '');
```

Comment:
Drop "Details: " from the start of the error message because it doesn't really make sense in the case of non-syntax error.